apps/g2-oidc/03-openstack/03-openstack.base/08-glance.yaml

```diff
@@ -0,0 +1,145 @@
+    targetPath: endpoints.oslo_messaging.auth.glance.password
+  # https://github.com/beskar-cloud/openstack-helm/blob/main/glance/values.yaml
+  values:
+    network:
+      api:
+        node_port:
+          enabled: true
+    images:
+      tags:
+        test: docker.io/xrally/xrally-openstack:2.0.0
+        glance_storage_init: docker.io/openstackhelm/ceph-config-helper:latest-ubuntu_xenial
+        glance_metadefs_load: &img_glance docker.io/openstackhelm/glance:yoga-ubuntu_focal-20230526
+        db_init: &img_heat docker.io/openstackhelm/heat:yoga-ubuntu_focal-20230526
+        glance_db_sync: *img_glance
+        db_drop: *img_heat
+        ks_user: *img_heat
+        ks_service: *img_heat
+        ks_endpoints: *img_heat
+        rabbit_init: docker.io/library/rabbitmq:3.9.29-management
+        glance_api: *img_glance
+        # Bootstrap image requires curl
+        bootstrap: *img_heat
+        dep_check: quay.io/airshipit/kubernetes-entrypoint:v1.0.0
+      pull_policy: Always
+    endpoints:
+      image:
+        scheme:
+          public: https
+        port:
+          api:
+            public: 443
+        hosts:
+          public: null
+        host_fqdn_override:
+          public: image.ostrava.openstack.cloud.dev
+      identity:
+        auth:
+          admin: {}
+            #password: ""          # secret/openstack/common/keystone.identity.admin.password
+            #region_name: ""       # configmap/common/region_name
+          glance: {}
+            #password: ""          # secret/openstack/common/glance.identity.glance.password
+            #region_name: ""       # configmap/common/region_name
+          test: {}
+            #password: ""          # secret/openstack/common/glance.identity.test.password
+            #region_name: ""       # configmap/common/region_name
+      oslo_db:
+        auth:
+          admin: {}
+            #password: ""          # secret/openstack/common/mariadb.admin.password
+          glance: {}
+            #password: ""          # secret/openstack/common/glance.oslo_db.glance.password
+      oslo_cache:
+        auth: {}
+          #memcache_secret_key: "" # secret/openstack/common/keystone.oslo_cache.memcache_secret_key
+      oslo_messaging:
+        auth:
+          admin:
+            username: rabbitmq-admin
+            #password: ""          # secret/openstack/common/rabbitmq.admin.password
+          glance:
+            username: glance-rabbitmq-user
+            #password: ""          # secret/openstack/common/glance.oslo_messaging.glance.password
+        statefulset:
+          replicas: 3
+          name: openstack-rabbitmq-rabbitmq
+    labels:
+      api: &common-labels
+        node_selector_key: openstack-control-plane
+        node_selector_value: enabled
+      job: *common-labels
+      registry: *common-labels
+    #pod:
+    #  replicas:
+    #    api: 3
+    #    registry: 2
+    # NOTE(portdirect): glance APIv1 is required for heat in Newton
+
```

Comment:
too many empty lines

Reply:
All unneeded empty lines in k8s/CRD manifests except empty lines in raw encoded config files were dropped by 04a287d016f855e488ed0d5ac6fc917c145f7c0c.